docs/conf.py

```diff
@@ -12,15 +12,16 @@
 
 import os
 import sys
-import sphinx_rtd_theme
 
-sys.path.insert(0, os.path.abspath('../'))
+# import sphinx_rtd_theme
```

Comment:
I thought the `import sphinx_rtd_theme` was needed for testing RTD builds?  I'm OK with leaving this as-is and seeing what happens when we check it in... if you're OK with me checking a fix into master if the doc build fails as a result.

Reply:
It would be interesting to see if it breaks. If it does, it is an easy fix. I will let you do the merge and check the docs to make sure they are correct or not.